src/test/crimson/CMakeLists.txt

```diff
@@ -8,9 +8,9 @@ add_executable(unittest_seastar_denc
 add_ceph_unittest(unittest_seastar_denc)
 target_link_libraries(unittest_seastar_denc crimson GTest::Main)
 
-#add_executable(unittest_seastar_messenger test_messenger.cc)
+add_executable(unittest_seastar_messenger test_messenger.cc)
```

Comment:
@tchaikov 
Changed the wrong commit titile from "enable unittest_seastar_denc" to "enable unittest_seastar_messenger"